scripts/vendorize_bencode.py

```diff
@@ -7,11 +7,15 @@
 import os
 import shutil
 import tarfile
-from StringIO import StringIO
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import BytesIO as StringIO
```

Comment:
I believe we can just use `from io import BytesIO` all the time here; Python 2.7 supports that.

Reply:
Sure, thanks for the refinement!